src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -57,6 +57,7 @@ private sealed partial class Emitter
             private const string JsonNamingPolicyTypeRef = "global::System.Text.Json.JsonNamingPolicy";
             private const string JsonSerializerTypeRef = "global::System.Text.Json.JsonSerializer";
             private const string JsonSerializerOptionsTypeRef = "global::System.Text.Json.JsonSerializerOptions";
+            private const string JsonSerializerContextTypeRef = "global::System.Text.Json.Serialization.JsonSerializerContext";
```

Comment:
Pretty surprised this constant doesn't exist already?

Reply:
Its defined for use in the Parser - I could pull the constant up into the `JsonSourceGenerator` itself if that would be preferred?